src/sth_configuration.js

```diff
@@ -12,17 +12,8 @@
   var dbPassword = ENV.DB_PASSWORD || config.database.password || '';
 
   module.exports = {
-    RANGE: {
-      YEAR: 'year',
-      MONTH: 'month',
-      WEEK: 'week',
-      DAY: 'day',
-      HOUR: 'hour',
-      MINUTE: 'minute'
-    },
     RESOLUTION: {
       MONTH: 'month',
-      WEEK: 'week',
```

Comment:
Basically because the standard time formats do not include this information and we would have to use some kind non-standard way of codifying it... I mean, regarding the `origin` value: `For certain resolution, it is the origin of time for which the aggregated time series information applies. For example, for a resolution of minutes, a valid origin value could be: 2015-03-01T13:00:00.000Z, meaning the 13th hour of March, the 3rd, 2015. The origin is stored using UTC time to avoid locale issues` :)